cpp/src/io/parquet/parquet.hpp

```diff
@@ -322,6 +322,15 @@ struct ColumnIndex {
   thrust::optional<std::vector<int64_t>> definition_level_histogram;
 };
 
+/**
+ * @brief Thrift-derived struct describing column sort order
+ */
+struct SortingColumn {
```

Comment:
Are we defining this twice? Why not use `sorting_column` from above?

Reply:
In general, we don't expose the thrift machinery outside libcudf. The file metadata, for instance, also has separate external definitions (like [here](https://github.com/rapidsai/cudf/blob/aab8a76b532b46713b9784302ffd202586ecb5cc/cpp/include/cudf/io/parquet_metadata.hpp#L60)). I'm just following that trend, even though it's duplicative. In this case, however, since the structs are truly identical, I could be convinced otherwise.

Reply:
Precedence is enough I believe.